src/game/boe.actions.cpp

```diff
@@ -3406,13 +3413,19 @@ void new_party() {
 	if(recording){
 		record_action("new_party", "");
 	}
-	if(overall_mode != MODE_STARTUP) {
-		std::string choice = cChoiceDlog("restart-game",{"okay","cancel"}).show();
+	if(party_in_memory) {
+		cChoiceDlog confirm("restart-game",{"okay","cancel"});
+		(confirm.operator->())->getControl("warning").replaceText("{{action}}", "Starting over");
```

Comment:
Huh? Wait. Isn't this `(confirm.operator->())->` just the same as `confirm->`? There's no reason to manually call the arrow operator here as far as I know…

Reply:
Yeah, this was very silly of me